jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -586,26 +585,21 @@
           (unless av
             (ros::ros-error "[:pick-object-with-movable-region] arm:~a IK before trying grasp fails. Abort picking" arm)
             (return-from :pick-object-with-movable-region :ik-failed)))
-        ;; Move whole arm to target pose
+        ;; Move whole arm to location above of the target pose
         (send *ri* :angle-vector-sequence-raw next-avs
               :fast (send *ri* :get-arm-controller arm :head t) 0 :scale 5.0)
         (send *ri* :wait-interpolation))
+      ;; Now hand is inside of the bin, so we can use cylindrical
+      (send *ri* :move-hand arm (send *baxter* :hand-grasp-pre-pose arm :cylindrical) 1000)
```

Comment:
Can you move this `:move-hand` just after https://github.com/start-jsk/jsk_apc/pull/2556/files/50baa62bd8009f2b4127c5fdd1afd4214e9468cd#diff-75b56ba9b0be842085e5a8a0c88f19f1L617 ?
After that `:wait-interpolation`, prismatic joint mainly moves, so we don't so worry about collision between finger and bin wall.